### ISSUE TITLE
Training code of stage 1

### ISSUE THREAD

**Issue by mniCJ** [JianghaiSCU/LightenDiffusion#16]
Dear authors,

Excellent work!

There is a problem I met when I tried to re-train your model, there seems no stage1 training code,  how to train with it?
Can you provide a training script for the stage1 model.

Looking forward to your reply.
Many thanks

**Comment by mniCJ** [comment]
After I trained 100k iter, the output during model verification was still noise, and the period loss function decreased normally. Do you know the possible reason.

Looking forward to your reply.
Many thanks!
![image](https://github.com/user-attachments/assets/7d1943fb-cee7-42fe-a803-8574e9151f9e)

**Comment by d-linlin** [comment]
> Dear authors,
> 
> Excellent work!
> 
> There is a problem I met when I tried to re-train your model, there seems no stage1 training code, how to train with it? Can you provide a training script for the stage1 model.
> 
> Looking forward to your reply. Many thanks

I have the same problem. How did you train the code for the first phase separately?

**Comment by Irving235** [comment]
你好，我跑代码的时候缺少一个unpaired_train.txt文件，请问一下可以在哪里能找到啊？

**Comment by mniCJ** [comment]
> > Dear authors,
> > Excellent work!
> > There is a problem I met when I tried to re-train your model, there seems no stage1 training code, how to train with it? Can you provide a training script for the stage1 model.
> > Looking forward to your reply. Many thanks
> 
> I have the same problem. How did you train the code for the first phase separately?-----------Deepl翻译-----------我也有同样的问题。您是如何单独训练第一阶段的代码的？

Same problem, I have not found a way to train one stage separately, hope the author can provide it. :D

**Comment by mniCJ** [comment]
> 你好，我跑代码的时候缺少一个unpaired_train.txt文件，请问一下可以在哪里能找到啊？
You can write your own script to generate the path, the format can be found in the previous closed problem.

**Comment by ZhanghansenIT** [comment]
> > 你好，我跑代码的时候缺少一个unpaired_train.txt文件，请问一下可以在哪里能找到啊？
> > You can write your own script to generate the path, the format can be found in the previous closed problem.

您好！我没有找到在哪里可以生成unpaired_train.txt，请问这个txt文件里面是什么内容呀。

**Comment by diwuqi** [comment]
你们训练集用的什么呀，训练集是非配对的低/高光照数据集，作者是不是没有提供呀

**Comment by ZhanghansenIT** [comment]
> After I trained 100k iter, the output during model verification was still noise, and the loss decreased normally. Do you know the possible reason.
> 
> Looking forward to your reply. Many thanks! ![image](https://private-user-images.githubusercontent.com/79388412/363618821-7d1943fb-cee7-42fe-a803-8574e9151f9e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU4NzE0NjYsIm5iZiI6MTcyNTg3MTE2NiwicGF0aCI6Ii83OTM4ODQxMi8zNjM2MTg4MjEtN2QxOTQzZmItY2VlNy00MmZlLWE4MDMtODU3NGU5MTUxZjllLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA5VDA4MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUxMzVmYzdiMDk2YzljNDhjOGEzMTNlOTg3YTQzMjYxYTk0ZDMwNjQ0M2EwMWM5MmEyY2UwNTI5M2NjNGE5M2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8CupvcXcugakxTAR77r0eK9I7bewjwgaCPfSUmqCQvM)
作者说训练 10w 迭代才有质变。
